### ISSUE TITLE
Compute Tc as a consistency check

### ISSUE THREAD

**Issue by sblunt** [sblunt/roman-orbits#1]
Since reported RVs collected from disparate sources may have, e.g. different negative sign conventions, we should think about consistency checks. Dmitry Savransky recommends computing Tc.

**Comment by sblunt** [comment]
Sort of related idea (not necessarily to check that the data are in the correct format, but to evaluate the orbit-fits themselves)-- we could try to systematically implement a cross-validation check

**Comment by zhexingli** [comment]
Are you referring to the different orbit conventions, e.g. opposite signs for omega, used in publications?

**Comment by sblunt** [comment]
I think the thing we’re worried about more is that some publications will define positive RV as moving toward us and some will define negative RV as moving away from us (and that will down the line result in the omega difference you mention, although that shouldn’t affect us directly)
(Also I’m just putting this here to remind myself to think about it btw, no action required imminently)

**Comment by zhexingli** [comment]
Hmm.. good point. If we have multiple datasets having different RV conventions and therefore not giving the same parameter values, mcmc would have a hard time converging anyway, no?

**Comment by sblunt** [comment]
Yes, absolutely agree that we’d get nonsensical/unconverged posteriors!
